aodndata/gsla/handler.py

```diff
@@ -13,35 +13,16 @@
 GSLA_REGEX = re.compile(r"""
                        IMOS_OceanCurrent_HV_
                        (?P<nc_time_cov_start>[0-9]{8}T[0-9]{6}Z)_GSLA_FV02_
-                       (?P<product_type>NRT00|DM00|DM01)_C-
-                       (?P<creation_date>[0-9]{8}T[0-9]{6}Z)\.nc(\.gz)?$
+                       (?P<product_type>NRT|DM02)\.nc(\.gz)?$
```

Comment:
i would imagine it would be better to have something such as
```python
(?P<product_type>NRT|DM0[1-2])\.nc(\.gz)?$
```
in order to allow multiple versions